client/src/components/InputFields/StatusRadioButtons.js

```diff
@@ -0,0 +1,110 @@
+import styled from '@emotion/styled';
+
+// STATUS BOX
+
+export const StatusWrapper = styled.div`
+  display: flex;
+  justify-content: space-around;
+  width: 98%;
+`;
+
+// STATUS ACTIVE
+
+export const ActiveLabel = styled.label`
```

Comment:
the label are mostly the same. Perhaps it is possible to style it once (width, fexbox, font-size...) and only create JSX components for the differences?